templates/poll_results.njk

```diff
@@ -57,6 +57,7 @@
 		.bar {
 			width: 5px;
 			height: 24px;
+			margin-left: 5px;
```

Comment:
![image](https://github.com/kgpmask/MASK/assets/42577624/ed93487e-a8a5-4d98-b7d6-bbbf542243e4)
This broke the results page and added huge bars

Reply:
![image](https://github.com/kgpmask/MASK/assets/42577624/2eb17eed-55ce-4f32-8cf0-b875eb1f4511)
Also needs padding at the bottom

Reply:
Padding in the footer or the result content?

Reply:
> ![image](https://user-images.githubusercontent.com/42577624/246351469-ed93487e-a8a5-4d98-b7d6-bbbf542243e4.png) This broke the results page and added huge bars

where is it broken ;-;

Reply:
> Padding in the footer or the result content?

Padding between the results and the footer?

Reply:
the white scroll bars in the side and at the bottom